src/rprblender/engine/viewport_engine_hybridpro.py

```diff
@@ -16,6 +16,7 @@
 from . import context_hybridpro
 
 import pyrpr
+import pyrprwrap
```

Comment:
no need to import pyrprwrap

Reply:
fixed https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/577/commits/bdb00ef37826dc9b26876222f8cd5e316a372e6b